api/main_endpoints/util/password.js

```diff
@@ -0,0 +1,52 @@
+const bcrypt = require('bcrypt');
+
+function validatePassword(password, strength = 'medium') {
```

Comment:
password strength functions already exist in [`api/main_endpoints/util/registerUser.js`](https://github.com/SCE-Development/Core-v4/blob/89a77d8ea0d8e5ab125fc26331fcdb4993352167/api/main_endpoints/util/registerUser.js)

you can add the hash password in there